src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/CallInstruction.cs

```diff
@@ -395,6 +395,8 @@ protected object[] GetArgs(InterpretedFrame frame, int first, int skip)
                 return Array.Empty<object>();
             }
         }
+
+        public override string ToString() => "Call(" + _target + ")";
```

Comment:
I'll keep it as-is for now; other instructions may benefit from similar changes, e.g. `ByRefNewInstruction`. We can make changes as we encounter some interpreter asserts that are unclear. Also, the by-ref nature is really a property of the target method or constructor, so it can be inferred from context (though the `ToString` of `MethodInfo` and `ConstructorInfo` could be insufficient to get the full details).